spec/lib/assessment_factory_spec.rb

```diff
@@ -282,6 +282,7 @@
       end
 
       describe "age range and subjects section" do
+        before { create(:country, code: "SG", subject_limited: true) }
```

Comment:
```suggestion
        before { create(:country, :requires_secondary_education_teaching_qualification, code: "SG") }
```

I think we should either use the trait, or remove the trait if we're not going to use it. If we rename the trait then we'll need to use the new name.